content/sensu-go/6.8/catalog/build-private-catalog.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Build a private catalog of Sensu integrations"
+linkTitle: "Build a Private Catalog"
+guide_title: "Build a private catalog of Sensu integrations"
+type: "guide"
+description: "Use Sensu's Catalog API to build and host your own private catalog of Sensu integrations."
+weight: 40
+version: "6.8"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.8:
+    parent: catalog
+---
+
+{{% notice commercial %}}
+**COMMERCIAL FEATURE**: Access the Sensu Catalog and integrations in the packaged Sensu Go distribution.
+For more information, read [Get started with commercial features](../../commercial/).
+{{% /notice %}}
+
+{{% notice note %}}
+**NOTE**: The Sensu Catalog is in public preview and is subject to change.
+{{% /notice %}}
+
+The [Sensu Catalog][1] is a collection of Sensu integrations that provide reference implementations for effective observability.
+The official Sensu Catalog is available in the [web UI][2], but you can also create a private catalog of custom integrations and make it available to users in the Sensu web UI.
+
+Before you begin, make sure that your integration files are saved in a repository that follows the required [organizational framework][3].
+
+Catalogs are namespaced, so you can have separate private catalogs with specific integrations for different groups of users.
+
+## Update URLs in integration asset builds
+
+If the assets for your private catalog are stored behind a firewall, update the asset definitions in your `sensu-resources.yaml` files to use the endpoint URL that will serve your catalog.
```

Comment:
```suggestion
If the assets for your private catalog are stored behind a firewall or are otherwise not publicly available, update the asset definitions in your `sensu-resources.yaml` files to use the endpoint URL that will serve your catalog.
```